apps/bfd-pipeline/bfd-pipeline-rda-bridge/src/main/java/gov/cms/bfd/pipeline/bridge/util/AttributionBuilder.java

```diff
@@ -1,47 +1,29 @@
 package gov.cms.bfd.pipeline.bridge.util;
 
-import java.io.BufferedReader;
+import freemarker.template.Configuration;
+import freemarker.template.Template;
+import freemarker.template.TemplateException;
+import freemarker.template.TemplateExceptionHandler;
```

Comment:
Freemarker is a good choice for a template engine.  There are probably others we could use but this one is fine.